### PR TITLE
Sh 104 dev

### PR DESCRIPTION
**Pull request by IPKIN9**
**Task Ticket**
SH104

**STAGES**
DEV: *
STG: 
PRD:

**Description**
First Upload Readme
**Screenshots**

### CODE REVIEW

File: README-wandi.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#LARAVEL#
```
Comment (sofyan48) [inline README-wandi.md]:
```suggestion
#LARAVEL
add full description 
```

Comment (IPKIN9) [reply README-wandi.md]: Sudah di ubah

File: README.md [hunk 2]
```diff
@@ -2,4 +2,7 @@
 Content management system Website Pena
 
 ## Getting Started
-Installing Laravel Version 6 And PHP FPM
\ No newline at end of file
+Installing Laravel Version 6 And PHP FPM
```
Comment (sofyan48) [inline README.md]: This is Not fully description

File: README-wandi.md (hunk 1 above)
Comment (sofyan48) [inline README-wandi.md]: Adding Getting Started for run this project

**Pull request merged**